src/java/com/sarality/app/view/list/ListViewInitializer.java

```diff
@@ -64,4 +65,8 @@ public void setupActions() {
       return new ListViewAdapter<T>(getContext(), dataList, getRowRenderer());
     }
   }
+
+  public ListViewAdapter<T> getAdapter() {
```

Comment:
The original question was "why" :)?

If we pass in the Filter to the initializer, why do we need the adapter. Just trying to see why we need access to Adapter and if there is some way to accomplish the same without exposing the Adapter.